include/mc_tasks/TrajectoryTaskGeneric.hpp

```diff
@@ -427,6 +427,7 @@ void TrajectoryTaskGeneric<T>::addToLogger(mc_rtc::Logger & logger)
 {
   logger.addLogEntry(name_ + "_damping", [this]() { return damping_(0); });
   logger.addLogEntry(name_ + "_stiffness", [this]() { return stiffness_(0); });
+  logger.addLogEntry(name_ + "_dimWeight", [this]() -> const Eigen::VectorXd { return dimWeight(); });
```

Comment:
Btw any particular reason why we don't have a `const-ref` accessor for `dimWeight`?